website/static/js/fileRevisions.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var ko = require('knockout');
+require('knockout-punches');
+var $ = require('jquery');
+var $osf = require('osfHelpers');
+var bootbox = require('bootbox');
+var waterbutler = require('waterbutler');
+
+ko.punches.enableAll();
+
+var Revision = function(data, file, node) {
+
+    var self = this;
+
+    $.extend(self, data);
+    var ops = {};
+    ops[self.versionIdentifier] = self.version;
+    self.downloadUrl = waterbutler.buildDownloadUrl(file.path, file.provider, node.id, ops);
+    self.date = new $osf.FormattableDate(data.modified);
+    self.displayDate = self.date.local !== 'Invalid date' ?
+        self.date.local :
+        data.date;
+
+    self.download = function() {
+        window.location = self.downloadUrl;
+        return false;
+    };
+
+};
+
+var RevisionsViewModel = function(node, file, editable) {
+
+    var self = this;
+
+    self.node = node;
+    self.file = file;
+    self.editable = editable;
+    self.urls = {
+        delete: waterbutler.buildDeleteUrl(file.path, file.provider, node.id),
+        download: waterbutler.buildDownloadUrl(file.path, file.provider, node.id),
+        revisions: waterbutler.buildRevisionsUrl(file.path, file.provider, node.id),
+    };
+    self.page = 0;
```

Comment:
Since we're not paginating revisions, can we delete all the associated code--references to `self.page`, `self.more`, etc?